frame/support/procedural/src/pallet/parse/config.rs

```diff
@@ -34,12 +34,16 @@ mod keyword {
 	syn::custom_keyword!(frame_system);
 	syn::custom_keyword!(disable_frame_system_supertrait_check);
 	syn::custom_keyword!(no_default);
+	syn::custom_keyword!(no_default_bounds);
 	syn::custom_keyword!(constant);
 }
 
 #[derive(Default)]
 pub struct DefaultTrait {
-	pub items: Vec<syn::TraitItem>,
+	/// A bool for each sub-trait item indicates whether the item has
+	/// `#[pallet::no_default_bounds]` attached to it. If true, the item will not have any bounds
+	/// in the generated default sub-trait.
+	pub items: Vec<(syn::TraitItem, bool)>,
```

Comment:
Can we use an enum instead of a boolean in here?

Reply:
I am not sure if it adds a lot of value here. What are the enum types that you would suggest?